src/ios/TLRKFeedback.m

```diff
@@ -34,6 +39,32 @@ -(void)initialize: (CDVInvokedUrlCommand *)command
         uid = objc_msgSend(objc_msgSend(UIDevice.currentDevice, identifierForVendorSEL), @selector(UUIDString));
     }
 
+   	// iOS localization specific options
+    // using UTF8 strings to allow displaying Chinese, Cyrillic, etc. chars 
+    NSDictionary *feedbackOptions = command.arguments.count > 2 ? command.arguments[2] : nil;
+
+    NSDictionary *iOSLocalizationOptions = [feedbackOptions valueForKey:kIOSFeedbackOptionsKeyName];
+	if(iOSLocalizationOptions != nil) {
```

Comment:
if the value for `kIOSFeedbackOptionsKeyName` is `nil`, then all `if` statements below will evaluate to `false` and the code won't be executed.